df.stockpile.xml

```diff
@@ -3,13 +3,13 @@
         <enum-item name='Any'/>
         <enum-item name='Stone'/>
         <enum-item name='Wood'/>
-        <enum-item name='Item'/>
-        <enum-item name='Bin'/>
         <enum-item name='Body'/>
         <enum-item name='Food'/>
```

Comment:
This is wrong, at least with respect to `world.stockpiles.num_haulers`, and I must recommend against this PR without more evidence.

I just tested on a mature forts by clearing all hauling labors on everyone. This resulted in `world.stockpiles.num_haulers` going to all zero. Then, reenabling food hauling on everyone resulted in `world.stockpiles.num_haulers[Food]` going to 194 while the others remained zero.

If the enum were wrong, some other field would have gone up. Therefore, the existing enum is correct, at least for food, but this change would change the value of the `Food` enum.

I must therefore object.